autoload/clap/themes/srcery.vim

```diff
@@ -0,0 +1,76 @@
+" Author: liuchengxu <xuliuchengxlc@gmail.com>
+" Description: Clap theme based on the material_design_dark theme.
+
+" Palette {{{
+
+let s:black          = ['#1C1B19', 0]
+let s:red            = ['#EF2F27', 1]
+let s:green          = ['#519F50', 2]
+let s:yellow         = ['#FBB829', 3]
+let s:blue           = ['#2C78BF', 4]
+let s:magenta        = ['#E02C6D', 5]
+let s:cyan           = ['#0AAEB3', 6]
+let s:white          = ['#D0BFA1', 7]
+let s:bright_black   = ['#918175', 8]
+let s:bright_red     = ['#F75341', 9]
+let s:bright_green   = ['#98BC37', 10]
+let s:bright_yellow  = ['#FED06E', 11]
+let s:bright_blue    = ['#68A8E4', 12]
+let s:bright_magenta = ['#FF5C8F', 13]
+let s:bright_cyan    = ['#53FDE9', 14]
+let s:bright_white   = ['#FCE8C3', 15]
+
+" xterm colors.
+let s:orange        = ['#FF5F00', 202]
+let s:bright_orange = ['#FF8700', 208]
+let s:hard_black    = ['#121212', 233]
+let s:xgray1        = ['#262626', 235]
+let s:xgray2        = ['#303030', 236]
+let s:xgray3        = ['#3A3A3A', 237]
+let s:xgray4        = ['#444444', 238]
+let s:xgray5        = ['#4E4E4E', 239]
+let s:xgray6        = ['#585858', 240]
+
+"}}}
+
+let s:save_cpo = &cpoptions
+set cpoptions&vim
+
+let s:palette = {}
+
+let s:palette.display = { 'ctermbg': s:xgray2[1], 'guibg': s:xgray2[0], 'ctermfg': 'NONE', }
+
+" Let ClapInput, ClapSpinner and ClapSearchText use the same backgound.
+let s:bg0 = { 'ctermbg': s:xgray5[1], 'guibg': s:xgray5[0] }
+let s:palette.input = s:bg0
+let s:palette.spinner = extend({ 'ctermfg': s:bright_orange[1], 'guifg': s:bright_orange[0], 'cterm': 'bold', 'gui': 'bold'}, s:bg0)
+let s:palette.search_text = extend({ 'ctermfg': s:bright_white[1], 'guifg': s:bright_white[0], 'cterm': 'bold', 'gui': 'bold' }, s:bg0)
+
+let s:palette.preview = { 'ctermbg': s:xgray4[1], 'guibg': s:xgray4[0] }
+
+let s:palette.selected = { 'ctermfg': 'NONE', 'guifg': s:bright_blue[0], 'ctermbg': s:xgray3[1] }
+let s:palette.current_selection = { 'ctermbg': s:xgray3[1], 'guibg': s:xgray3[0], 'cterm': 'bold', 'gui': 'bold' }
+
+let s:palette.selected_sign = { 'ctermfg': s:green[1], 'guifg': s:green[0] }
+let s:palette.current_selection_sign = { 'ctermfg': s:red[1], 'guifg': s:red[0] }
+
+" Initial attempt at something
+" TODO: Check for transparent variable maybe?
+function! ClapSrceryLoad() abort
+	highlight SignColumn ctermbg=NONE guibg=NONE
+endfunction
+
+function! ClapSrceryUnload() abort
+	highlight SignColumn ctermbg=0 guibg=#1C1B19
+endfunction
+
+augroup ClapSrcery
```

Comment:
@roosta Your thoughts on something like this?  Seems to work for the non-trans version at least.

Reply:
Looks good! non-trans?

Reply:
Non-transparent.

Reply:
oh, should probably have figured that out :P